heroku/cli
# chore(certs): add code coverage reports

## Why the change?
[GUS Card](https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07EE00001NC2GbYAL/view)

As per our review of unit tests and coverage for the certs package, this PR does the following:
- Cleans up test coverage report ✅


## Additional Notes
- Removed `all` property in `.nycrc` file to remove `src/index.ts` from testing since there is nothing to test in that file
- No actual tests to implement as nothing exists in the command. The only test is an no op `it` block.

## How to verify?
1. Pull down branch
2. Run tests
3. Confirm test coverage report is visible.
